assets/src/design-system/components/input/index.js

```diff
@@ -157,29 +152,14 @@ export const Input = forwardRef(
     const inputRef = useRef(null);
     const [focused, setFocused] = useState(false);
 
-    useEffect(() => {
```

Comment:
@BrittanyIRL I made some changes here and pulled the parts that can be shared between the inputs out to `utils`. LMK if you see any concerns with that.

Mixing TextArea with Input didn't seem a very "clean" solutions, however, there are parts that can still be shared.

Reply:
that makes sense! Tagged @samwhale for his opinion here too since he authored this originally.

Reply:
I like what you did with this ✨